css-viewport/Overview.bs

```diff
@@ -260,6 +261,85 @@ as follows:
 
 Issue: Specify extend-to-zoom behavior by the viewport meta tag
 
+<h3 id="interactive-widgets-section">''interactive-widgets''</h3>
+
+Issue: Move the definition of ''visual viewport'' from CSSOM-View to this spec.
+
+The <dfn><code>interactive-widgets</code></dfn> property specifies the effect that interactive UI
+widgets have on the page's viewports. It defines whether widgets overlay a given viewport or whether
+the viewport is shrunken so that it remains fully visible while the widget is showing. Interactive
+UI widgets are transient user agent or operating system UI through which a user can provide input.
+
+<div class="note">The most common such UI widget is a virtual keyboard.</div>
+
+The following is a list of valid values for [=interactive-widgets=] and the associated
+viewport-resizing behavior:
+
+<dl>
+	<dt><dfn><code>overlays-content</code></dfn></dt>
+	<dd>
+		Interactive UI widgets MUST NOT [=resize=] the [=initial viewport=] nor
+		the <a spec="CSSOM-VIEW">visual viewport</a>. The user agent must perform the same steps
+		<!--TODO I can't seem to autolink to overlaysContent, help?-->
```

Comment:
@tabatkins can you help with this?

Reply:
Got some help with this internally - added an anchors block at the top so autolinks to `overlaysContent` are working now.

Reply:
Can i follow!!